packages/web-components/src/components.d.ts

```diff
@@ -551,7 +551,7 @@ export namespace Components {
         /**
           * Whether or not an analytics event will be fired.
          */
-        "enableAnalytics": boolean;
+        "enableAnalytics"?: boolean;
```

Comment:
Majority of the Type Definition fixes seen in this file are because of changing "required" props into "optional" props. They all have defaults assigned to them in their respective component files, so making them optional doesn't change any base functionality.